src/model/build.jl

```diff
@@ -21,6 +21,8 @@ Arguments
 - `optimizer`:
     the optimizer factory that should be attached to this model (e.g. Cbc.Optimizer).
     If not provided, no optimizer will be attached.
+- `formulation`:
+    the details of which constraints, variables, etc. to use
```

Comment:
This change seems unrelated from the previous one (adding reserve shortfalls). For this PR that's fine, but in general it's considered a best practice to only include related changes within each commit or PR. This makes it much easier later on to know which commit or PR introduced a bug.